### ISSUE TITLE
HTTPS for openra.net

### ISSUE THREAD

**Issue by xan2622** [OpenRA/OpenRAWeb#372]
> Starting in July, Google Chrome will mark all HTTP sites as “not secure".

https://security.googleblog.com/2018/02/a-secure-web-is-here-to-stay.html
https://nakedsecurity.sophos.com/2018/01/17/firefox-locks-down-its-future-with-https-secure-contexts/

It might be interesting to move http://www.openra.net to https as well.
See also: https://letsencrypt.org

**Comment by Echolon** [comment]
Yes, just opened an issue for the same reason at the wrong place :/

**Comment by pchote** [comment]
The main website is hosted on Github Pages, which doesn't support https for custom domains.

**Comment by Echolon** [comment]
Support said they will make a +1 in their internal Feature Request List and alternatively named this link https://blog.cloudflare.com/secure-and-fast-github-pages-with-cloudflare/

But I think this is not of your interest....
Guess we need to wait for it :/

**Comment by pchote** [comment]
Github have finally announced support for https on custom domains: https://blog.github.com/2018-05-01-github-pages-custom-domains-https/

https://www.openra.net/ already kind of works, and we can look into fixing the remaining issues and setting up the http -> https redirects when time allows.

**Comment by Echolon** [comment]
Ouh yeah, I made it :)

**Comment by pchote** [comment]
I expect https://github.com/OpenRA/OpenRA/pull/15519 should fix the asset downloads, so we should be able to reenable this after the next release.

**Comment by Echolon** [comment]
You're still my beloved mod project :smiley:

**Comment by anjew175** [comment]
I assume this issue is fixed?

**Comment by Echolon** [comment]
Not forced
